include/micm/process/cuda_process_set.hpp

```diff
@@ -31,7 +31,7 @@ namespace micm
     void SetJacobianFlatIds(const SparseMatrix<double, OrderingPolicy>& matrix);
 
     template<template<class> typename MatrixPolicy>
-    requires VectorizableDense<MatrixPolicy<double>> std::chrono::nanoseconds AddForcingTerms(
+    requires VectorizableDense<MatrixPolicy<double>> void AddForcingTerms(
         const MatrixPolicy<double>& rate_constants,
         const MatrixPolicy<double>& state_variables,
         MatrixPolicy<double>& forcing)
```

Comment:
```suggestion
    requires VectorizableDense<MatrixPolicy<double>> void AddForcingTerms(
        const MatrixPolicy<double>& rate_constants,
        const MatrixPolicy<double>& state_variables,
        MatrixPolicy<double>& forcing) const
```

Reply:
I think this is a Github issue. I have added the `const` back already (line 38).

Reply:
ah, sorry - I missed that